package.json

```diff
@@ -121,7 +121,7 @@
 		"babel-plugin-require-context-hook": "1.0.0",
 		"benchmark": "2.1.4",
 		"browserslist": "4.9.1",
-		"chalk": "^2.4.2",
+		"chalk": "3.0.0",
```

Comment:
The earlier version shipped with types that weren't compatible in a CommonJS setting.

See: https://github.com/chalk/chalk/pull/344

Release notes for the major version: https://github.com/chalk/chalk/releases/tag/v3.0.0

I couldn't find any instances of `chalk.enabled` or `chalk.constructor` that would be problematic for an upgrade.

Reply:
Can we update `chalk` also for all WordPress packages? I see the old version used in the lock file. We could avoid installing local copies of `chalk` for several packages this way.

Reply:
I had seen the other `package-lock.json` changes, but assumed they were from other dependencies, not our own packages. I can see about updating those as well 👍

Reply:
Updated in aac72435c3.

There's been another major release in the time since the pull request was opened:

https://github.com/chalk/chalk/releases/tag/v4.0.0

As best I can tell, we're not using Level, and already require Node 10+, so it shouldn't be an issue for us. I've updated this in the root `package.json` as well.